src/styles.css

```diff
@@ -54,26 +48,33 @@ body {
 /* NAV BAR */
 nav {
   display: flex;
-  justify-content: flex-end;
-  align-items: flex-start;
-  width: 100%;
+  justify-content: space-between;
+}
+
+nav > h1 {
+  text-align: center;
+  margin-left: 25px;
+  cursor: pointer;
 }
 
 .nav-child {
   display: flex;
```

Comment:
Do you think we could give some spacing between home icon, searchbar, search icon? It would look super clean.

Reply:
done